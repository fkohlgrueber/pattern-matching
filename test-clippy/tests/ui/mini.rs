fn main() {
    let x = if true {
        true;
        'x'
    } else {
        'y'
    };

    let y = if true {
        'x'
    } else {
        'y'
    };
}
