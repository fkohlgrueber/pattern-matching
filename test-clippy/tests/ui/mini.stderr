error: This is a match for the mini pattern. Well Done too!
  --> $DIR/mini.rs:2:13
   |
LL |       let x = if true {
   |  _____________^
LL | |         true;
LL | |         'x'
LL | |     } else {
LL | |         'y'
LL | |     };
   | |_____^
   |
   = note: `#[forbid(mini_pattern)]` on by default

error: This is a match for the mini pattern. Well Done too!
  --> $DIR/mini.rs:9:13
   |
LL |       let y = if true {
   |  _____________^
LL | |         'x'
LL | |     } else {
LL | |         'y'
LL | |     };
   | |_____^

error: aborting due to 2 previous errors

